mdflynn/whats-cookin
# Feature/connect users to modal buttons

#### What's this PR do?
* Connect the user to the modal view and modal buttons

#### Where should the reviewer start?
* Follow the green lines in `scripts.js`

#### How should this be manually tested?
* Pull down, deploy the site, and for a specific recipe try the star button or cook button
  * In console type `currentUser.recipiesToCook` or `currentUser.favoriteRecipes` and notice how the array will contain the spcific recipe info.

#### Any background context you want to provide?
* Look into how onchange HTML elements work
* Iterator methods used

#### What are the relevant tickets?
N/A
